03.bowling/bowling.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+score = ARGV[0]
+scores = score.split(',')
+shots = scores.map { |s| s == 'X' ? 10 : s.to_i }
```

Comment:
```suggestion
shots = ARGV[0].split(',').map { |s| s == 'X' ? 10 : s.to_i }
```

賛否両論ありそうですが、メソッドチェーンでひとまとめにしてしまう方法もあります